docs/general.md

```diff
@@ -0,0 +1,101 @@
+# `Simulation` base class
+
+We defined a base class named `Simulation`. You can find its Python schema definition in [src/nomad_simulations/general.py](https://github.com/nomad-coe/nomad-schema-plugin-simulation-data/blob/develop/src/nomad_simulations/general.py#L168). In NOMAD, this section will
+appear under the **_data_** section for the _archive_ of each _entry_.
+
+In NOMAD, a set of [base sections](https://nomad-lab.eu/prod/v1/staging/docs/howto/customization/base_sections.html) derived from the [Basic Formal Ontology (BFO)](https://basic-formal-ontology.org/) are defined. We used them to define `Simulation` as an [`Activity`](http://purl.obolibrary.org/obo/BFO_0000015). The UML diagram is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/simulation_base.drawio.png" alt="Simulation base section diagram." width="80%" title="Click to zoom in">
```

Comment:
What does the "s" in the diagrams following the type mean? I.e., `np.float64, s`? It is not immediately clear to me.

Reply:
These are the units. Yeah, I agree it is pretty obscure. Should I remove them or come up with a more clear naming?

Reply:
Added a note